governance/votes/Executive vote - April 1, 2022.md

```diff
@@ -0,0 +1,87 @@
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+- A total of 2,433,452.30 DAI (itemized further down) will be transferred to multiple Core Units.
+- A total of 6,866,436.69 DAI (itemized further down) will be streamed to multiple Core Units.
+- The existing DAI budget stream for the Real-World Finance Core Unit (RWF-001) will be revoked.
+- Three Core Unit vesting streams for a total of 167.64 MKR (itemized further down) will be set up.
+- A total of 60 MKR will be transferred to Long For Wisdom as retroactive compensation.
+- A total of 731,000 DAI will be streamed to the Gelato Keeper Network.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Core Unit Budget Transfers
+
+As per multiple successful budget proposals (linked below), a total of **2,433,452.30 DAI** will be transferred to multiple Core Units if this executive proposal passes:
+
+- 30,000 DAI will be transferred to the [GovAlpha Core Unit (GOV-001)](https://mips.makerdao.com/mips/details/MIP39c2SP3) at 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73 as a result of [their most recent budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP59).
+- 348,452.30 DAI will be transferred to the [Immunefi Security Core Unit (IS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP24) at 0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4 as a result of [an amendment to their MIP40c3-SP42 budget](https://mips.makerdao.com/mips/details/MIP4c2SP17).
+- 2,055,000 DAI will be transferred to the [Real-World Finance Core Unit (RWF-001](https://mips.makerdao.com/mips/details/MIP39c2SP24) at 0x96d7b01Cc25B141520C717fa369844d34FF116ec as a result of [their most recent budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP61).
```

Comment:
We've been linking to the addresses on etherscan recently, might be something to add here